Teacher login

andisaidheyyeayeayea opened the pull request:
Renamed dolandev branch to Teacher login

Commit:
initial commit for teacher login [branch ch55]

Commit:
initial commit for teacher login [branch ch55]

andisaidheyyeayeayea committed:
Add .gitignore (David put this in here with his permission)

andisaidheyyeayeayea committed:
Add logintest.html, set up directories for project

andisaidheyyeayeayea committed:
Check to see if clubhouse is connected to github [ch55]

andisaidheyyeayeayea committed:
As a teacher I want to be able to login so [ch55]

andisaidheyyeayeayea committed:
google show login, gconnect and gdisconnect functions completed, but untested.

jd12 reviewed (commented):
Nice initial commit. I would add some comments to the gconnect to explain what each part is doing.

andisaidheyyeayeayea committed:
fix: Server will now actually run and route to a homepage, but homepage is not found D:

andisaidheyyeayeayea committed:
Fix: Login function works! You should be able to login with a google account.

andisaidheyyeayeayea committed:
Add file

andisaidheyyeayeayea committed:
Deleted unnecessary file

andisaidheyyeayeayea committed:
Fix: logintest syntax errors fixed, however still getting an error 500 internal server error.

andisaidheyyeayeayea committed:
Fix: Google login kinda working but not really (html is working as intended)

andisaidheyyeayeayea committed:
Fix: Google login for both chadiwkc and regular emails working, typeerror returning for some reason.

andisaidheyyeayeayea committed:
Fix: Google Login now fully functional (I think)

andisaidheyyeayeayea committed:
added test main page

andisaidheyyeayeayea committed:
Added a failed login page, not sure if it works.

andisaidheyyeayeayea committed:
No change to functionality

andisaidheyyeayeayea committed:
Fix: Syntax error, website now loads properly

andisaidheyyeayeayea committed:
Added: Create teacher function that currently serves no purpose

andisaidheyyeayeayea committed:
Added: Not working teacher-student differentiator (the code will probably make you vomit)

andisaidheyyeayeayea committed:
Fix: fixed teacher-student identification system but it still doesn't work i'm saying fix because this is how DB wants us to format our commits

andisaidheyyeayeayea committed:
Fix: Logging in will now return that you have failed to log in. always.

andisaidheyyeayeayea committed:
Feat: Created a createuser page and function, however this will likely be removed in favor of more secure methods of database population.

andisaidheyyeayeayea committed:
Add: Created a database populator to test the functionality of the database and login functions

andisaidheyyeayeayea committed:
Fix: Didn't actually fix anything but trying a new way of differentiating between teacher and student.

Commit:
Fix: Fix displaying teacher page when teacher logs in vs student login

You need to use one_or_none() when calling the database. Then a slight
tweak to your logged in method. The framework was already there though.

andisaidheyyeayeayea committed:
Fix: Teacher Login and Student login REALLY work now its not a lie

andisaidheyyeayeayea committed:
Feat (?): Added Create Student method that will add a student if their username is not already in the database but I'm not really sure if this actually work (NEEDS TESTING)

andisaidheyyeayeayea committed:
Fix: Login now properly adds new students to the database.
